src/asynchronous/mod.rs

```diff
@@ -16,8 +16,9 @@ pub mod shutdown;
 mod unix_incoming;
 
 pub use self::stream::{
-    ClientStream, ClientStreamReceiver, ClientStreamSender, Kind, ServerStream,
```

Comment:
`Kind` is dropped here.  Is that intentional?

Reply:
Thanks, my fault, fixed.